tests/socket/socket_tests.cpp

```diff
@@ -32,7 +32,7 @@ connection_test(
 {
     struct bpf_object* object;
     int program_fd;
-    int result = bpf_prog_load("cgroup_sock_addr.o", BPF_PROG_TYPE_CGROUP_SOCK_ADDR, &object, &program_fd);
+    int result = bpf_prog_load_deprecated("cgroup_sock_addr.o", BPF_PROG_TYPE_CGROUP_SOCK_ADDR, &object, &program_fd);
```

Comment:
Why not make changes to use the non-deprecated API?

Reply:
That would be a larger change, the deprecated API is still supported to ease portability of older apps.
So it is worth continuing to test it for now.

Reply:
Can you then at least make the changes in this test code? We can create an issue to delete the deprecated API.

Reply:
#1146